SharePointFramework/PortfolioWebParts/src/models/ProjectListModel.ts

```diff
@@ -28,6 +31,9 @@ export class ProjectListModel {
     public groupId: string,
     public title: string,
     public url: string,
+    public GtProjectLifecycleStatus: string, 
+    public GtProjectServiceAreaText: string,
+    public GtProjectTypeText: string,
```

Comment:
Don't use the full field name.

Use `type`.

Reply:
Type already exists, we can use `projectType`?

Reply:
What is the existing `type` used for then?

Reply:
It seems like it is being used within the ProjectTimeline

Reply:
@siifux That's weird :D As it's named ProjectListModel since it's used for the ProjectList webpart (which you're working on now).

Reply:
@Remi749 Do you know what the **type** property is used for in ProjectTimeline?

Reply:
I had to go down memory lane here. But if I recall correctly, *type* in the ProjectListModel that was introduced in the ProjectTimeline was to seperate projects from timeline elements (phase, subphases, milestones). 

![image](https://user-images.githubusercontent.com/28678468/161711125-4055f610-04d7-49b3-ace2-1f6a61b5cd62.png)

As you can see, it's hardcoded. I know back then I had some problems seperating projects, phases, subphases and milestone. I don't think I actually need it anymore if it doesn't belong. 

Thoughts?

Reply:
I'll look into it @Remi749. Thinks got rather confusing when model `ProjectListModel` got used in `ProjectTimeline` component.

Reply:
I'm going to make some changes to the Timeline, so I could in my future PR remove the need for the fields that were added to the ProjectListModel that has to do with the Timeline